pyproject.toml

```diff
@@ -57,7 +57,8 @@ myst-parser = {version = ">=0.16.1"}
 camelot = "camelot.__main__:main"
 
 [tool.poetry.group.base.dependencies]
-ghostscript = "^0.7"
+pypdfium2 = ">=4,<5"
+ghostscript = "^0.7"  # remove in favor of pypdfium2?
 opencv-python = "^4.7.0.68"
```

Comment:
The `^` specifier here is new to me and [not listed in this doc](https://pip.pypa.io/en/stable/reference/requirement-specifiers/#examples), what does it mean? Is it similar to `~=`, maybe?

Reply:
Poetry doesn't follow the pep here: https://python-poetry.org/docs/dependency-specification/

Reply:
Ah, I see (I'm not used to poetry). So IIUC `pypdfium2 = ">=4,<5"` is the same as `pypdfium2 = "^4"`.

Reply:
Hmm the test failures are simply because pypdfium2 is not present in the test env.
Apparently the entry above is not sufficient? Could you give some direction where we need to add pypdfium2?

Reply:
Ah probably in the noxfile.